src/sentry/static/sentry/app/views/alerts/wizard/options.tsx

```diff
@@ -39,3 +40,37 @@ export const AlertWizardDescriptions: Record<AlertType, string> = {
   trans_duration: t('Alert on the duration of transactions'),
   lcp: t('Alert on longest contentful paint'),
 };
+
+export type WizardRuleTemplate = {
+  aggregate: string;
+  dataset: Dataset;
+  eventTypes: EventTypes;
+};
+
+export const AlertWizardRuleTemplates: Partial<Record<AlertType, WizardRuleTemplate>> = {
```

Comment:
Partial here feels weird maybe Exclude the one key instead? can go either way i guess

Reply:
ahh yea forgot about exclude, will change